src/game.cpp

```diff
@@ -4109,7 +4109,11 @@ void game::monmove()
         }
 
         m.creature_in_field( critter );
-
+        if( calendar::once_every( 1_days ) ) {
+            critter.refill_udders();
+            critter.try_biosignature();
+            critter.try_reproduce();
```

Comment:
I agree, this is more realistic, but it would also drastically increase the amount of crap if player lingers around for a while (sleeping, crafting). There's already an issue with tedious cleanup (mentioned in #24294). It can become worse.

Reply:
The amount of crap would be the same as if they had walked away from their farm and then back again, it wouldnt be any more than the usual poop catchup because itd update the biosig timers etc .
whether that same amount of poop appears in real-time or when you come back from a journey shouldnt really change how annoying it is?

Reply:
Fair enough.